### PR TITLE
fix: update to latest terraform module

### PR DESCRIPTION
**Pull request by peter-poki**

### CODE REVIEW

**Comment by rawlingsj** [comment]
great thanks - I'm not that familiar with AKS, I wonder if @hervelemeur you could take a look at this PR when you get a min?

File: main.tf [hunk 1]
```diff
@@ -30,6 +28,10 @@ module "cluster" {
   subnet_cidr                      = var.subnet_cidr
   subnet_name                      = var.subnet_name
   vnet_cidr                        = var.vnet_cidr
+  
+  registry_resource_group_name         = var.registry_resource_group_name
+  use_existing_acr_name                = var.use_existing_acr_name
+  use_existing_acr_resource_group_name = var.use_existing_acr_resource_group_name
```
Comment (hervelemeur) [inline main.tf:34]:
```suggestion
  registry_resource_group_name         = var.registry_resource_group_name
  use_existing_acr_name                = var.use_existing_acr_name
  use_existing_acr_resource_group_name = var.use_existing_acr_resource_group_name
```
Also, all "-" should be with the same indent.

**Review by hervelemeur — CHANGES_REQUESTED** [review]

**Review by hervelemeur — APPROVED** [review]

**Pull request merged**